shared/templates/audit_rules_privileged_commands/ansible.template

```diff
@@ -26,6 +26,24 @@
       - "{{ find_{{{ NAME }}}.files | map(attribute='path') | list | first }}"
```

Comment:
These changes are present in #6635 https://github.com/ComplianceAsCode/content/pull/6635/files#diff-e73c037376dc344836ec83f2504b924a7e76ec61ca03d613194b2464638ab77e

Can you remove them from here?

Reply:
yes